module/Photo/src/Mapper/Tag.php

```diff
@@ -44,6 +45,32 @@ public function getTagsByLidnr(int $lidnr): array
         );
     }
 
+    /**
+     * Get all the tags for a photo, but limited to lidnr and full name.
+     *
+     * @param int $photoId
+     * @return array
+     */
+    public function getTagsByPhoto(int $photoId): array
+    {
+        $rsm = new ResultSetMapping();
+        $rsm->addScalarResult('id', 'id', 'integer')
+            ->addScalarResult('lidnr', 'lidnr', 'integer')
+            ->addScalarResult('fullName', 'fullName');
+
+        $sql = <<<QUERY
```

Comment:
Why don't we use the query builder here?

Reply:
One of the main reasons is speed, loading the complete `Member` entity for just full name (for which we also have to call the function) brings a lot of overhead from Doctrine. So if we directly want to get the full name from the query (and preventing us from loading the whole entity) we would have to use `CONCAT_WS`. Unfortunately, this is not supported in the query builder/DQL and hence I used a native query here.